code/tools/pkgresources/autorun_launchd_script/autorun.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+/bin/rm /Users/Shared/.com.googlecode.munki.autorunmanagedsoftwarecenter
```

Comment:
What the heck is this referring to?

Reply:
Ups sorry, I left things I used to test this in my local env